src/ServiceManager/Travely.ServiceManager.DAL/DbContextOptionsBuilderExtension.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Travely.ServiceManager.DAL
+{
+    public static class DbContextOptionsBuilderExtension
+    {
+        public static void UseCustomDatabaseServer(this DbContextOptionsBuilder options, IConfiguration configuration)
```

Comment:
Please use more descriptive name for a method, maybe `UseServiceManagerDatabaseServer`